renderer/src/components/Totals/Totals.tsx

```diff
@@ -123,9 +140,44 @@ const Totals = ({ selectedDate }) => {
     }
   };
 
+  const changePeriod = (periodName, selectedDate) => {
+    let dateName = "";
+
+    switch (periodName) {
+      case "day":
```

Comment:
And create a new constant for day, week, today, etc. If it's already existed then use it.
`const DATE_PERIODS = { DAY: 'day' ... };`